samples/msal-node-samples/auth-code-cli-app/src/index.js

```diff
@@ -52,8 +52,8 @@ acquireToken().then((response) => {
     callMicrosoftGraph(response.accessToken).then((graphResponse) => {
         console.log(graphResponse);
         process.exit(0);
-    }).catch((e) => {
-        console.log(e);
+    }).catch((error) => {
+        console.error(error);
```

Comment:
error.errorMessage

Reply:
No, because the function this calls already returns error.errorMessage

Reply:
Never assume an error will match a given shape.